pandas/io/formats/style.py

```diff
@@ -483,6 +486,12 @@ def to_latex(
             the left, centrally, or at the right.
         siunitx : bool, default False
             Set to ``True`` to structure LaTeX compatible with the {siunitx} package.
+        environment : str, optional
+            If given, the environment that will replace 'table' in ``\\begin{table}``.
+            If 'longtable' is specified then a more suitable template is
```

Comment:
small comment here. can you list the valid values? (or provide a link to them). can be a followon.